library/src/main/java/com/yayandroid/locationmanager/providers/locationprovider/DispatcherLocationProvider.java

```diff
@@ -155,14 +155,14 @@ void askForGooglePlayServices(int gpServicesAvailability) {
     void resolveGooglePlayServices(int gpServicesAvailability) {
         LogUtils.logI("Asking user to handle GooglePlayServices error...");
         gpServicesDialog = getSourceProvider().getGoogleApiErrorDialog(getActivity(), gpServicesAvailability,
-              RequestCode.GOOGLE_PLAY_SERVICES, new DialogInterface.OnCancelListener() {
-                  @Override
-                  public void onCancel(DialogInterface dialog) {
-                      LogUtils.logI("GooglePlayServices error could've been resolved, "
-                            + "but user canceled it.");
-                      continueWithDefaultProviders();
-                  }
-              });
+                RequestCode.GOOGLE_PLAY_SERVICES, new DialogInterface.OnCancelListener() {
```

Comment:
Can you please clean up these spaces since there is no actual change here this file should not be in this PR?